src/components/Cluster/ClusterDetail/NodePool.tsx

```diff
@@ -35,12 +38,23 @@ const NPViewAndEditNameStyled = styled<
   }
 `;
 
+interface INodesWrapperProps {
+  highlight?: boolean;
+  children: React.ReactNode;
+}
+
 const NodesWrapper = styled.div`
   width: 36px;
   height: 30px;
   line-height: 31px;
   text-align: center;
   border-radius: 3px;
+  white-space: nowrap;
+  ${({ highlight }: INodesWrapperProps) =>
+    highlight &&
+    css`
+      background-color: ${theme.colors.goldBackground};
+    `}
```

Comment:
This should also work:
```suggestion
  background-color: ${({ theme, highlight }) => highlight && theme.colors.goldBackground};
```

Reply:
Whats the type of the parameter of this function then (`{ theme, highlight }`)?

Reply:
Probably something like `INodesWrapperProps | undefined`, but I remember that we decided to not type styled components

It's also written here, under the `Style` section
https://github.com/giantswarm/happa/pull/1268